app/models/middleware_messaging.rb

```diff
@@ -0,0 +1,6 @@
+class MiddlewareMessaging < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
```

Comment:
I think in the Domain mode this may not be true, as a destination is set up on the level of a profile, which is associated with the server group and thus applies to all servers in that server group. 

I think reporting it for each server in a server group may be ok, as effectively they would share the same destination and either use it for load balancing or such.